### ISSUE TITLE
broken modified run_classifier.py link for hyperdrive

### ISSUE THREAD

**Issue by sriramjcet** [microsoft/AzureML-BERT#51]
Hi,
I am trying to follow the tutorial to finetune BERT and  perform hyperparamter tuning. The instruction refers to adding logging of metrics to run_classifier.py and refers to the modified run_classifier.py in hyperlink. But that is broken. Can you please share the modified run_classifier.py training script that has logging included for hyperdrive? Appreciate the help. Copy pasted the instructions from the tutorial below

Further within run_classifier.py, we log the learning rate, and the epoch training and eval loss the model achieves:

run.log('lr', np.float(args.learning_rate))

run.log('train_mean_loss', mean_loss)
run.log('eval_mean_loss', mean_loss)
run.log('train_example_loss', mean_loss)
run.log('eval_example_loss', mean_loss)
These run metrics will become particularly important when we begin hyperparameter tuning our model in the "Tune model hyperparameters" section.

**Let's first copy the modified run_classifier.py into our local project directory** this link is broken
Thanks,
Sriram
